journald/journald.cpp

```diff
@@ -75,6 +83,27 @@ class JournaldLoggerProcess : public Process<JournaldLoggerProcess>
       std::strcpy((char*) entries[i].iov_base, entry.c_str());
     }
 
+    if (flags.destination_type != "journald") {
```

Comment:
Just for the record, adding a third option sounds excessive (7 more configurations to test/support!).  

If not for backwards compatibility, we'd only have two options (`sandbox` or `journald`) (not `both`).  And this would be achieved entirely via configuration.  Future destination types should not require changes to this module.